src/components/metadata/styles/metadata-code.scss

```diff
@@ -0,0 +1,182 @@
+@import '../../../styles/variables';
+@import '../../../styles/extends';
+@import '../../../styles/mixins';
+@import './metadata-code-themes.scss';
+
+.kui-theme--light {
+  --color-metadata-code-bg: #{$color-bg-light-4};
+}
+
+.kui-theme--dark {
+  --color-metadata-code-bg: #{$color-bg-dark-3};
+}
+
+.pipeline-metadata-code {
+  @extend %sidebar;
+
+  $margin: 24px;
+
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: -1px; /* Avoids pixel rounding gaps */
+  right: 0;
+  bottom: -1px;
+
+  // On small screens anchor to the left side of the screen
+  left: 0;
+
+  z-index: 2;
+  padding: 0 $metadata-sidebar-width-open 0 0;
+  background: var(--color-metadata-code-bg);
+  transform: translateX(100vw);
+  transition: transform ease 0.5s 0.1s, left ease 0.5s;
+
+  &--visible {
+    transform: translateX(0);
+  }
+
+  @mixin breakpoints($sidebar-width) {
+    // When screen is the minimum usable width
+    @media (min-width: $metadata-sidebar-code-breakpoint) {
+      // Anchor code to the left sidebar
+      left: $sidebar-width + $margin;
+    }
+
+    // When screen is wide enough to fit all sidebars
+    $sidebar-total-width: $sidebar-width + $margin + $metadata-sidebar-code-open +
+      $metadata-sidebar-width-open;
+    @media (min-width: #{$sidebar-total-width}) {
+      // Anchor to right sidebar
+      left: calc(
+        100% - #{$margin + $metadata-sidebar-code-open +
+          $metadata-sidebar-width-open}
+      );
+    }
+  }
+
+  // When sidebar is open
+  &--sidebarVisible {
+    @include breakpoints($sidebar-width-open);
+  }
+
+  // When sidebar is closed
+  &--no-sidebarVisible {
+    @include breakpoints($sidebar-width-closed);
+  }
+
+  @media print {
+    display: none;
+  }
+}
+
+.pipeline-metadata-code__title {
+  margin: 2.1em 36px 1.8em 36px;
+  flex-grow: 0;
+  font-weight: normal;
+  font-size: 1.7em;
+  line-height: 1.6;
+}
+
+.pipeline-metadata-code__code {
+  display: block;
+  flex-grow: 1;
+  overflow: auto;
+  color: var(--color-text);
```

Comment:
You've removed the new variable but left the declaration - adding `color: var(--color-text);` seems unnecessary to me because that's already the default color for the application so it just overrides the default value?

Reply:
Ah I misread your earlier comment and thought that was the suggested change, I've removed this line.